caip2.md

```diff
@@ -0,0 +1,129 @@
+---
+namespace-identifier: icp-caip2
+title: ICP Namespace - Chains
+author: Quint Daenen (@q-uint)
+discussions-to: TODO
+status: Draft
+type: Standard
+created: 2023-05-17
+requires: CAIP-2
+---
+
+# CAIP-2
+
+*For context, see the [CAIP-2][] specification.*
+
+## Rationale
+
+The CAIP-2 namespace for the Internet Computer Protocol (ICP) provides a standardized system for identifying assets
+within the network. It promotes interoperability, efficient querying, and clarity in asset identification, benefiting
+developers and users working with decentralized applications and assets on the Internet Computer.
+
+## Syntax
+
+The ICP namespace is `icp` and the reference is the first 32 bytes of the SHA-256 hash of the root key of the ICP instance.
```

Comment:
```suggestion
The ICP namespace is `icp` and the reference is the SHA-256 hash of the root key of the ICP instance.
```

Reply:
Thanks for catching this! The reference is limited to `reference: [-_a-zA-Z0-9]{1,32}` so it only fits half of the hash in hexadecimal format.